<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBOp_Personenkunde</class>
 <widget class="QDialog" name="DBOp_Personenkunde">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personenkunde</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>442</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPersonenkundenID">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PersonenkundenID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="linePersonenkundenID">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelStandortID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>StandortID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboStandortID">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushNeuS">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Neu...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAnrede">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anrede</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboAnrede">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Herr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Frau</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelVorname">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vorname</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineVorname">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelNachname">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nachname</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineNachname">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelPosition">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="linePosition">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelAbteilung">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Abteilung</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineAbteilung">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelGebaeude">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Geb√§ude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineGebaeude">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelZimmer">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zimmer</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineZimmer">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelTelefon">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefon</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineTelefon">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelTelefax">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefax</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineTelefax">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>eMail</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEmail">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DBOp_Personenkunde</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DBOp_Personenkunde</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
